dashboard/lib/policies/lti.rb

```diff
@@ -51,6 +51,8 @@ module AccessTokenScopes
     },
   }
 
+  MAX_COURSE_MEMBERSHIP = 500
```

Comment:
Max number of students we want to sync for a course. 500 seemed reasonable as an upper bound since the largest we've seen so far is 100ish. Open to changing if we know of a scenario for larger courses.

Reply:
What's the experience if one were to have over the max?   Any helpful messaging?

Reply:
Right now, no helpful messaging - they would just get the max number of students synced, and the rest would appear to be missing. One teacher in the pilot program already experienced this with her 100-person course